### PR TITLE
WIP: Basic abstract stuff

### PR DESCRIPTION
**Pull request by ThiefMaster**
Just to make looking at and reviewing the changes easier! :smirk_cat:

### CODE REVIEW

File: indico/modules/events/abstracts/__init__.py [hunk 1]
```diff
@@ -32,6 +34,26 @@
...
+
+@signals.event.get_feature_definitions.connect
+def _get_feature_definitions(sender, **kwargs):
+    return AbstractsFeature
+
+
+class AbstractsFeature(EventFeature):
+    name = 'abstracts'
+    friendly_name = _('Abstracts')
+    description = _('Gives event managers the opportunity to do a "Call for Abstracts" and use the abstract '
```
Comment (nop33) [inline indico/modules/events/abstracts/__init__.py]: I would rephrase it to 
> Gives event managers the opportunity to **open** a "Call for Abstracts

sounds better imo

File: indico/modules/events/features/controllers.py [hunk 2]
```diff
@@ -37,7 +38,6 @@
 class RHFeaturesBase(RHConferenceModifBase):
     def _checkParams(self, params):
```
Comment (nop33) [inline indico/modules/events/features/controllers.py]: Looks like this method is not needed since all it does is to call the parent's class method

Comment (ThiefMaster) [inline indico/modules/events/features/controllers.py]: Ah right, didn't look closer when removing the `self.event` assignment
